src/modules/Authentication.js

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios';
+import store from '../state/store/configureStore';
+import errorHandler from './ErrorHandler';
+
+const Authentication = {
+  async login(event) {
+    let params = {
+      email: event.target.email.value,
+      password: event.target.password.value,
+    };
+
+    axios
+      .post('auth/sign_in', params)
+      .then((createResponse) => {
```

Comment:
why do you use then and not saving response into a variable like we usually do?

Reply:
Not really sure :P